src/include/stir/scatter/ScatterEstimation.h

```diff
@@ -170,6 +171,11 @@ class ScatterEstimation : public ParsingObject
   void set_recompute_mask_image(bool arg);
   void set_recompute_mask_projdata(bool arg);
 
+  //! Sets the forward projector for the scatter estimation
+  void set_forward_projector_sptr(const shared_ptr<ForwardProjectorByBin> projector_sptr);
+  //!Get the forward projector used for the scatter estimation
+  shared_ptr<ForwardProjectorByBin> get_forward_projector_sptr() const;
```

Comment:
need to change naming. These are currently only used for the sino mask generation. Maybe `....forward_projector_for_mask...`?